va/va.h

```diff
@@ -256,6 +256,13 @@ typedef int VAStatus;	/** Return status type from functions */
 #define VA_FILTER_SCALING_NL_ANAMORPHIC 0x00000300
 #define VA_FILTER_SCALING_MASK          0x00000f00
 
+/** Interpolation method for scaling */
+#define VA_FILTER_INTERPOLATION_DEFAULT                    0x00000000
```

Comment:
should you add some comments here? https://github.com/intel/libva/blob/master/va/va_vpp.h#L971

Reply:
What kinds of comments? Here, I have document that for interpolation method.

Reply:
you will use it just like https://github.com/intel/libva/blob/master/va/va_vpp.h#L971, right?

Reply:
@XinfengZhang yes, I agreed with you. Will add some comments in filter_flags.

Reply:
Thanks @XinfengZhang for the comments. Updated.